asreview/json_logging.py

```diff
@@ -0,0 +1,238 @@
+import json
+from collections import OrderedDict
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+
+import asreview
+from asreview.settings import ASReviewSettings
+from asreview.analysis.statistics import _get_labeled_order
+from asreview.analysis.statistics import _get_last_proba_order
+
+
+class Dict_Logger(object):
+    """Class for logging the Systematic Review with no storage."""
+    def __init__(self, *_, **__):
```

Comment:
A general remark. Think about the structure of the Class. This syntax is a bit odd. 

```
class A(object)
    version = "1.0"
    read_only = True
    def __init__(self):
        pass
```
or 
```
class A(object)
    def __init__(self, version="1.0", read_only=True):
         self.version = version
         self.read_only = read_only
```

Or a combination of both